sympy/core/containers.py

```diff
@@ -231,8 +231,9 @@ def __new__(cls, *args):
             items = [Tuple(k, v) for k, v in args]
         else:
             raise TypeError('Pass Dict args as Dict((k1, v1), ...) or Dict({k1: v1, ...})')
+        items = tuple(sorted(items, key=default_sort_key))
```

Comment:
I think that `ordered` might be more efficient than directly using `sort` with `default_sort_key`.

Reply:
I am not familiar with what you are refering to with `ordered`

Reply:
https://github.com/sympy/sympy/blob/5dc2df6aa7b68d42d39ab59a1b6b50f40ebebd19/sympy/core/sorting.py#L201-L210